### PR TITLE
Centering Items using CSS Flexbox #6906

### PR DESCRIPTION
**Pull request by khush2706**
<hr>

## Description 📜

Added the documentation for centering items in CSS using flexbox

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by khush2706** [comment]
Can someone please review my PR..?

File: Web_Development/FrontEnd/Centering items using CSS Flexbox/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      }
+```
+
+![align-items](assets/column2.png)
+
+Reference-links:
+
+1) https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox
+2)  https://www.w3schools.com/css/css3_flexbox.asp
+3) https://www.better.dev/centering-things-with-css-flexbox
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/Centering items using CSS Flexbox/README.md:129]: Write the name of the website and embed the link in it.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add image source, just below it.
Remove all the binary files, just drag and drop images into md file to add them.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add image source below every image and embed the link in name of the website.

**Comment by khush2706** [comment]
> Add image source below every image and embed the link in name of the website.
All the images I've used are screenshots except one for which I've added the source below it..

**Comment by kashishahuja2002** [comment]
> > Add image source below every image and embed the link in name of the website.
> > All the images I've used are screenshots except one for which I've added the source below it..

ok

File: Web_Development/FrontEnd/Centering items using CSS Flexbox/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+
+## flex-direction property
+
+By default the flex-direction is set as row. If we change the flex-direction to column that would interchange the main-axis and the cross-axis. It will also make our items stack in a column instead of a row like before.
+
+![flex-direction](https://user-images.githubusercontent.com/86849664/139647286-56adde37-e110-407f-b338-30754270d768.png)
+
+<p align="center"> Image Source: https://miro.medium.com/max/700/1*bw4hK0MYxFXbw7ylW2fR5Q.png </p>
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/Centering items using CSS Flexbox/README.md:97]: Embed the link in the website name.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

File: Web_Development/FrontEnd/Centering items using CSS Flexbox/README.md (hunk 2 above)
Comment (khush2706) [reply Web_Development/FrontEnd/Centering items using CSS Flexbox/README.md]: @kashishahuja2002 I've made the changes pls check.

**Review by kashishahuja2002 — APPROVED** [review]
LGTM. Tag the supervisor of your batch to merge the PR.

**Comment by khush2706** [comment]
> LGTM. Tag the supervisor of your batch to merge the PR.

k. Thank you so much!

**Comment by khush2706** [comment]
> LGTM. Tag the supervisor of your batch to merge the PR.

@padmajabhol Can you please merge my PR.

**Comment by Knighthawk-Leo** [comment]
Great work @khush2706 ✨

**Comment by khush2706** [comment]
> Great work @khush2706 ✨

Thank you so much!😊

**Pull request merged**